api-datos/cmd/tablacruce/modelos.go

```diff
@@ -33,11 +38,15 @@ type CountTableRow struct {
 	Codigo        string
 }
 
-// NewIsland Crea una nueva estructura de tipo Island
-func NewIsland(id int, nombre string, longitud, latitud float64) Island {
-	return Island{id, nombre, longitud, latitud}
-}
-
 func NewCountTableRow(nombreEspecie string, nombreIsla string, maximoNidos int, temporada int, latitud, longitud float64, codigo string) CountTableRow {
```

Comment:
Parece que falta agregar el tipo de la variable `latitud`:
```suggestion
func NewCountTableRow(nombreEspecie string, nombreIsla string, maximoNidos int, temporada int, latitud float64, longitud float64, codigo string) CountTableRow {
```